mindjunkies/forums/views.py

```
@@ -145,29 +149,91 @@ def get(self, request, *args, **kwargs):
             "reply": reply,
-            "replyForm": reply_form,
+            "replyForm": replyForm,
         }
         return render(request, "forums/reply_form.html", context)
+    
+class LikeToggleView(LoginRequiredMixin, View):
+    """Base view for toggling likes on objects"""
+    model = None
+    template_name = None
+    context_object_name = None
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)